docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+Makes speaking certain languages a prerequisite.
+
+**Examples:**
+
+`     PRELANG:1,Dwarven,Elven    `
+
+Character must be able to speak either "Dwarven" or "Elven".
+
+`     PRELANG:2,Dwarven,Elven    `
+
+Character must be able to speak both "Dwarven" and "Elven".
+
+`     PRELANG:2,Dwarven,Elven,Halfling    `
+
+Character must be able to speak any two of "Dwarven", "Elven" or "Halfling".
+
+`     PRELANG:3,ANY    `
+
+Character must be able to speak any three languages.
+
+`     PRELANG:4,TYPE=Spoken    `
+
+Character must be able to speak any four languages that can be spoken.
+
+------------------------------------------------------------------------
+
+### PRELEGS
+
+Requires a creature to have a certain number of legs.
+
+See also <a href="#prehands">PREHANDS</a>.
+
+#### Syntax
+
+`     PRELEGSoperator:number_of_legs    `
+
+`     operator    ` is a comparison operator from the list `     EQ    ` , `     LT    ` , `     LTEQ    ` , `     GT    ` , `     GTEQ    ` , or `     NEQ    ` . The comparison operator is written as part of the tag name, i.e. `     PRELEGSLTEQ    ` . See [comparison operators](#comparison-operators) for more details.
+
+The `     operator    ` is written as part of the tag name, i.e. `     PRELEGSNEQ    ` or `     PRELEGSGTEQ    ` .
+
+`number_of_legs` is the required number of legs.
+
+#### Examples
+
+`     PRELEGSGTEQ:4    `
+
+Character must have at least 4 legs.
+
+------------------------------------------------------------------------
+
+<span id="prelevel"></span> \*\*\* Updated 5.13.5
+
+**Tag Name:** PRELEVEL:x,y
+
+**Variables Used (x):** MIN=Number or formula (Minimum total level).
+
+**Variables Used (y):** MAX=Number or formula (Maximum total level).
+
+**What it does:**
+
+-   Provides the minimum and/or maximum total levels, including all character, racial, or monster class levels, the character must have to qualify.
+-   Either value may be used without the other.
+
+**Example:**
+
+`     PRELEVEL:MIN=5    `
+
+Character must be at least 5th level.
+
+`     PRELEVEL:MAX=5    `
+
+Character must be no greater than 5th level.
+
+`     PRELEVEL:MIN=5,MAX=10    `
+
+Character must be at least 5th level but no greater than 10th level.
+
+`     !PRELEVEL:MIN=3,MAX=6    `
+
+Character must NOT be 3rd, 4th, 5th or 6th level.
+
+------------------------------------------------------------------------
+
+**<span id="prelevelmax"></span> Tag Name:** PRELEVELMAX:x
+
+**Variables Used (x):** Number (The maximum level).
+
+**What it does:**
+
+Requires that a character have a maximum number of character (non-monster) levels.
+
+-- Do NOT mix PRELEVELMAX and BONUS:SKILLPOINTS|NUMBER, since it results in double negative and some very bad exceptions in the code.
+
+**Example:**
+
+`     PRELEVELMAX:10    `
+
+Character cannot be over level 10.
+
+------------------------------------------------------------------------
+
+<span id="premove"></span> \*\*\* Updated 5.9.4
+
+**Tag Name:** PREMOVE:x,y=z,y=z
```

Comment:
This is a "feature" request - PREMOVE will only evaluate movement modes granted by MOVE, it ignores movement derived completely by BONUS:MOVEADD - not sure how you would note that.